### PR TITLE
Adding Okta Jenkins from mesos-systemd

### PR DESCRIPTION
**Pull request by matthewdfuller**
* Migrating PR from here: https://github.com/adobe-platform/mesos-systemd/pull/175/

### CODE REVIEW

File: v1/util/jenkins-update-config.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+if [[ ! $1 ]]; then
+  echo "$USAGE_MESSAGE"
+  exit 1;
+fi
+
+CONTROL_JENKINS_OKTA_METADATA=$(etcdctl get /jenkins/config/okta/metadata)
+CONTROL_JENKINS_ADMIN_GROUP=$(etcdctl get /jenkins/config/okta/admin-group)
+CONTROL_JENKINS_RO_GROUP=$(etcdctl get /jenkins/config/okta/read-group)
+
+if [[ -n $CONTROL_JENKINS_OKTA_METADATA ]] ; then
```
Comment (matthewdfuller) [inline v1/util/jenkins-update-config.sh]: May want to consider checking all of these:
```
CONTROL_JENKINS_OKTA_METADATA
CONTROL_JENKINS_ADMIN_GROUP
CONTROL_JENKINS_RO_GROUP
```

File: v1/util/jenkins-update-config.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+JENKINS_DIRECTORY=$1
+
+USAGE_MESSAGE="Control Jenkins: Please provide the Jenkins home directory"
+
+if [[ ! $1 ]]; then
+  echo "$USAGE_MESSAGE"
+  exit 1;
+fi
+
+CONTROL_JENKINS_OKTA_METADATA=$(etcdctl get /jenkins/config/okta/metadata)
```
Comment (chr0n1x) [inline v1/util/jenkins-update-config.sh]: @matthewdfuller would it be possible to use that `etcd-get` function that you wrote? I'm wary of any  that touches the FS now.

Comment (matthewdfuller) [inline v1/util/jenkins-update-config.sh]: This intentionally might not exist though (if the user wants to use insecure Jenkins)

**Comment by chr0n1x** [comment]
👍

**Pull request merged**